test/Interop/Cxx/stdlib/use-std-iterator.swift

```diff
@@ -1,37 +0,0 @@
-// RUN: %target-run-simple-swift(-I %S/Inputs -Xfrontend -enable-experimental-cxx-interop)
```

Comment:
This is not something that we want people to be able to do anymore. The correct way to do this is with the stdlib overlay, which we already have tests for.